### PR TITLE
Changes to observable mathml expressions

### PR DESCRIPTION
**Pull request by jClugstor**
adresses https://github.com/DARPA-ASKEM/simulation-integration/issues/46

### CODE REVIEW

**Comment by fivegrant** [comment]
@jClugstor can you post the results of the dashboard?

**Comment by jClugstor** [comment]
PyCiemms

image

SciML

image

File: data/models/SEIRHD_NPI_Type1_petrinet.json [hunk 1]
```diff
@@ -217,7 +217,7 @@
     {
      "target": "I",
      "expression": "I0",
-     "expression_mathml": "<cn>I0</cn>"
+     "expression_mathml": "<ci>I0</c>"
```
Comment (fivegrant) [inline data/models/SEIRHD_NPI_Type1_petrinet.json:220]:
```suggestion
     "expression_mathml": "<ci>I0</ci>"
```

**Review by fivegrant — COMMENTED** [review]
Looks good to me except for the tiny suggestion I had. 

I'll merge if @sabinala approves since this changes a lot of the models she created.

**Comment by sabinala** [comment]
Looks good to me, I committed those changes

**Pull request merged**